__tests__/e2e/import-library/import-library.test.ts

```diff
@@ -0,0 +1,12 @@
+// @ts-ignore
+import { ContentfulClient } from '@onebeyond/agnostic-cms-harmonizer';
+
+it('should check the imported library is an instance of ContentfulClient', () => {
+  const instance = new ContentfulClient({
+    accessToken: '',
+    space: '',
+    environment: '',
+  });
+  expect(instance).toBeInstanceOf(ContentfulClient);
+});
+describe('import-library', () => {});
```

Comment:
Just out of curiosity, what does this describe do? Why do we need it if it's empty?

Reply:
It does nothing xD
I will remove it